.gitignore

```diff
@@ -1,2 +1,9 @@
+#ignore all python cached files
 __pycache__/
-.vscode/*
\ No newline at end of file
+
+#ignore all vscode settings
+.vscode/
+
+#ignore all data from coverage.py
+.coverage
+htmlcov/
```

Comment:
Good idea, thank you.  Having quantitative code coverage will help us in creating needed tests.